packages/@nucleus/app/styles/doc/_playgroud.scss

```diff
@@ -0,0 +1,50 @@
+$playground-border-color: #dae1e7;
+
+.playground-container {
+  display: flex;
+  border-bottom: 1px dotted $playground-border-color;
+  padding: 10px;
+  justify-content: space-around;
+
+  &--component {
+    align-self: center;
+    width: 50%;
+    text-align: center;
```

Comment:
This might affect the childrens' text-align and get applied to nucleus-components. Needs some looking into